File: Dataset/Leetcode/train/48/460.py
 class Solution:
    def XXX(self, matrix: List[List[int]]) -> None:
        matrix[:] = matrix[::-1]
        hang = len(matrix)
        for i in range(hang):
            for j in range(hang):
                if i > j:
                    matrix[i][j],matrix[j][i] = matrix[j][i], matrix[i][j]
        return matrix

File: Dataset/Leetcode/valid/7/460.cpp
 class Solution {
public:
    int XXX(int x) {
        int pos = 0;
        string max;
        string x_str;
        string temp;
        istringstream iss;
        ostringstream oss;
    
        oss << INT_MAX;
        max = oss.str();
        oss.str("");

        oss << x;
        x_str = oss.str();
        if (x < 0) pos++;
        std::XXX(x_str.begin() + pos, x_str.end());
        temp = x_str.substr(pos);
        if (temp.length() == max.length() && temp > max) return 0;
        iss.str(x_str);
        iss >> x;
        return x;
    }
};

